api/marketdata.go

```diff
@@ -160,6 +160,18 @@ func getChartsHandler(charts *market.Charts, cache *caching.Provider) func(c *gi
 		}
 		token := c.Query("token")
 
+		coinObj, ok := coin.Coins[uint(coinId)]
+		if !ok {
+			c.JSON(http.StatusOK, watchmarket.ChartData{})
+			return
```

Comment:
I would move this into separate function, get ticker by coin and token. 

in that case there is only one function to return `c.JSON(http.StatusOK, watchmarket.ChartData{})`

Reply:
Can you check new commit? Do I get your point right?

Reply:
`getTickersHandler` is a rest API handler. 
it should:
- route request to the proper service

it should avoid to:
- contain business logic

Reply:
I thought about creating additional service and do all the action there, but currently it is not implemented at any handler

We will find a better solution, I created an issue for that https://github.com/trustwallet/watchmarket/issues/99